### PR TITLE
Add a setting for the Server Switch notification

### LINKED ISSUE #778: Option to disable switching servers notification
There is no option to disable notifications for switching servers. These notifications don't seem necessary when the user is manually switching servers in the UI. They would be helpful if the VPN switched servers due to some network error.

Found by: cpeterson

### PR DESCRIPTION
**Pull request by strseb**
Closes #778

### CODE REVIEW

**Review by bakulf — CHANGES_REQUESTED** [review]
Do you mind extending the functional tests to cover this PR too?
There is a setting test. All the rest is good to me! I haven't reviewed the QML part.

**Comment by strseb** [comment]
@bakulf tests are now green :)

File: src/inspector/inspectorwebsocketconnection.cpp [hunk 1]
```diff
@@ -162,6 +162,18 @@ static QList<WebSocketSettingCommand> s_settingCommands{
           return SettingsHolder::instance()->localNetworkAccess() ? "true"
                                                                   : "false";
         }},
+    // local area network access
```
Comment (bakulf) [inline src/inspector/inspectorwebsocketconnection.cpp:165]: wrong title

File: tests/functional/testServers.js [hunk 2]
```diff
@@ -162,6 +162,7 @@ describe('Server list', function() {
   });
 
   it('server switching', async () => {
+    await vpn.setSetting('server-switch-notification', 'true');
```
Comment (bakulf) [inline tests/functional/testServers.js:165]: Good! Can you also add 1 line in testSettings.js to test the UI  of this new checkbox?

**Review by bakulf — CHANGES_REQUESTED** [review]

File: tests/functional/testServers.js (hunk 2 above)
Comment (strseb) [reply tests/functional/testServers.js:165]: Added :)

**Review by bakulf — APPROVED** [review]

**Comment by bakulf** [comment]
Can you please rebase the patch and maybe add the checkbox for all the kinds of notifications? connection/disconnection/switching?

**Comment by strseb** [comment]
@bakulf - updated the PR, also now has a switch for connected/disconnected notifications

closes #930

**Review by bakulf — APPROVED** [review]

File: src/ui/settings/ViewNotifications.qml [hunk 3]
```diff
@@ -29,61 +29,100 @@ VPNFlickable {
...
+        VPNCheckBoxRow {
+            id: switchServersAlert
+            objectName: "switchServersAlert"
+            visible: VPNFeatureList.notificationControlSupported
+
+            width: parent.width
+
+            //% "Server Switchting Notification"
+            labelText: qsTrId("vpn.settings.notification.serverSwitch")
+            //% "Get notified when you successfully switched Servers"
```
Comment (flodolo) [inline src/ui/settings/ViewNotifications.qml:99]: Why is "Servers" capitalized here?

File: src/ui/settings/ViewNotifications.qml [hunk 4]
```diff
@@ -29,61 +29,100 @@ VPNFlickable {
...
+        VPNCheckBoxRow {
+            id: connectionChangeAlert
+            objectName: "connectionChangeAlert"
+            visible: VPNFeatureList.notificationControlSupported
+
+            width: parent.width
+
+            //% "Connection Change Notification"
+            labelText: qsTrId("vpn.settings.notification.connectionChange")
+            //% "Get notified when the Connection Status Changed"
```
Comment (flodolo) [inline src/ui/settings/ViewNotifications.qml:117]: The capitalization all over the place here. _Changed_ should not be capitalized, and AFAIK there's no feature called "Connection Status", so I'm not sure why that's capitalized either.

**Comment by bakulf** [comment]
I created a new PR to fix these strings.

**Pull request merged**